api/src/main/java/uk/gov/dhsc/htbhf/claimant/converter/ClaimToClaimResponseDTOConverter.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.dhsc.htbhf.claimant.converter;
+
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Component;
+import uk.gov.dhsc.htbhf.claimant.entity.Claim;
+import uk.gov.dhsc.htbhf.claimant.model.ClaimResponseDTO;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+@Component
+@AllArgsConstructor
+public class ClaimToClaimResponseDTOConverter {
+
+    public List<ClaimResponseDTO> convert(List<Claim> claims) {
+        List<ClaimResponseDTO> listOfClaims = new ArrayList<>();
+        for (Claim claim : claims) {
+            listOfClaims.add(ClaimResponseDTO.builder()
+                    .id(claim.getId())
+                    .claimStatus(claim.getClaimStatus())
+                    .firstName(claim.getClaimant().getFirstName())
```

Comment:
local variable for claim.getClaimant() as used multiple times

Reply:
and also worth extracting ClaimResponseDTO builder in separate method